portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -14,6 +14,13 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.data.Comment;
```

Comment:
order of the imports should be alphabetical. Also I would put this line together with the one above.